src/Compilers/CSharp/Portable/BoundTree/BoundExpression.cs

```diff
@@ -48,6 +48,12 @@ public virtual bool SuppressVirtualCalls
                 return false;
             }
         }
+
+        public new Nullability TopLevelNullability
```

Comment:
>TopLevelNullability [](start = 31, length = 19)

Should we call this `Nullability` rather than `TopLevelNullability`? `Nullability` seems unambiguous.

Reply:
I specifically called this `TopLevelNullability` because I don't think `Nullability` is unambiguous.

---
In reply to: [252397266](https://github.com/dotnet/roslyn/pull/32899#discussion_r252397266) [](ancestors = 252397266)